### PR TITLE
Added : all pages for video library and basic routing

### PR DESCRIPTION
**Pull request by rutvikpumak**
-> Basic pages routes added
-> Added
 1) Auth Page
 2) Liked Page
 3) Watch Later Page
 4) Playlist Page
 5) History Page

### CODE REVIEW

**Review by aniketxparihar — APPROVED** [review]
Code looks clean and didn't find any issue. Good to go!

File: src/App.js [hunk 1]
```diff
@@ -1,18 +1,35 @@
...
   return (
     <div className="App">
-      <Navbar />
       <Router>
+        <Navbar />
         <Sidebar />
         <Routes>
           <Route path="/" element={<VideoListing />} />
           <Route path="/singleVideo" element={<SingleVideo />} />
+          <Route path="/playlist" element={<Playlist />} />
```
Comment (xsaurabhsuthar) [inline src/App.js:25]: There's a repetition of route

File: src/App.js [hunk 2]
```diff
@@ -1,18 +1,35 @@
...
+          <Route path="/playlist" element={<Playlist />} />
+          <Route path="/playlist/playlistList" element={<PlaylistList />} />
```
Comment (xsaurabhsuthar) [inline src/App.js:26]: Can be named better

File: src/pages/Auth/Login.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  return (
+    <div className="auth-container flex-center">
+      <div className="auth-main-container flex-center">
+        <div className="auth-title">
+          <h2 className="text-center">Login</h2>
+        </div>
+        <form className="auth-main" onSubmit={(e) => e.preventDefault()}>
+          <div className="auth-email">
+            <label htmlFor="mail">Email Address</label>
+            <input
```
Comment (xsaurabhsuthar) [inline src/pages/Auth/Login.js:15]: input can be converted to a new component as it's getting repeated.

**Comment by xsaurabhsuthar** [comment]
- Good directory structure.
- Clean code
- Good variable names.
Good to merge 🚀🚀🚀🚀

**Review by xsaurabhsuthar — APPROVED** [review]

**Pull request merged**